runtime/compiler/runtime/J9Profiler.cpp

```diff
@@ -1643,7 +1643,9 @@ TR_BlockFrequencyInfo::TR_BlockFrequencyInfo(TR_CallSiteInfo *callSiteInfo, int3
    _blocks(blocks),
    _frequencies(frequencies),
    _counterDerivationInfo(NULL),
-   _entryBlockNumber(-1)
+   _entryBlockNumber(-1),
+   _isQueuedForRecompilation(0),
```

Comment:
I'm guessing there is no easy existing flag for this? If there isn't why is it on the block info and not the body info? This bit will be stale if kept on the blockinfo post recompilation because we keep the last good blockinfo structure around until the next profiling structure is populated with enough data.

Reply:
I checked into PersistentMethodBodyInfo and PersistentJittedBodyInfo. Could not find any flag that says this guy is being recompiled now.
When implementing this facility, I found two possible candidates where we can put this information. One is BlockFrequencyInfo and one is PersistentJittedInfo.
I wanted to have it stored in Class that is related to profiling. If I use persistentJittedInfo then I would have increases a size for all methods. Consumer of this field is limited to profiling compilations using jProfiling (JProfilingValue, JProfilingBlock and JProfilingRecompLoopTest) generates test to check if we have queued for recompilation in profiling compilations using jProfiling. So I chose to put this in TR_BlockFrequencyInfo.